shared/constants/network.ts

```diff
@@ -704,9 +705,10 @@ export const TEST_CHAINS: Hex[] = [
   CHAIN_IDS.MONAD_TESTNET,
 ];
 
-export const CAIP_FORMATTED_EVM_TEST_CHAINS = TEST_CHAINS.map(
-  (chainId) => `eip155:${hexToNumber(chainId)}`,
-);
+export const CAIP_FORMATTED_TEST_CHAINS = [
```

Comment:
For more type-safety we could also use `CaipChainId[]` here too? I've checked `NON_EVM_TESTNET_IDS` and it's already using this type, but enforcing the type sounds more future-proof to me, WDYT?